strongbox-client/src/test/java/org/carlspring/strongbox/service/impl/ProxyRepositoryConnectionPoolConfigurationServiceImplIT.java

```diff
@@ -1,5 +1,7 @@
 package org.carlspring.strongbox.service.impl;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I think you can safely revert this change. It just re-orders the import and it's not according to our convention.

Reply:
Done.